projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -69,7 +69,10 @@ public function reregister_menu_items() {
 		parent::reregister_menu_items();
 
 		$this->add_my_home_menu();
-		$this->add_inbox_menu();
+
+		if ( ! get_option( 'is_staging_site' ) ) {
```

Comment:
This option will be blown away if the user uploads a database.

Let's use Atomic Persistent Data instead, and reference a `WPCOM_IS_STAGING_SITE` bit.